MatrixSDK/Utils/Media/MXMediaLoader.m

```diff
@@ -232,6 +233,14 @@ - (void)connection:(NSURLConnection *)connection willSendRequestForAuthenticatio
     NSURLProtectionSpace *protectionSpace = [challenge protectionSpace];
     if ([protectionSpace.authenticationMethod isEqualToString:NSURLAuthenticationMethodServerTrust])
     {
+        NSSet <NSData *> *certificates = [AFSecurityPolicy certificatesInBundle:[NSBundle mainBundle]];
+        if (certificates && certificates.count > 0) {
```

Comment:
For code consistency, could you please add a new line before the opening bracket?